executor/src/witgen/jit/symbolic_expression.rs

```diff
@@ -55,6 +58,25 @@ pub enum UnaryOperator {
     Neg,
 }
 
+impl<T: FieldElement, S> Children<SymbolicExpression<T, S>> for SymbolicExpression<T, S> {
```

Comment:
Not sure if it is worth implementing this trait (especially since we don't implement `children_mut()`...), but I thought it might be useful again.